modules/osdk-updating-v125-to-v128.adoc

```diff
@@ -0,0 +1,154 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/golang/osdk-golang-updating-projects.adoc
+// * operators/operator_sdk/ansible/osdk-ansible-updating-projects.adoc
+// * operators/operator_sdk/helm/osdk-helm-updating-projects.adoc
+// * operators/operator_sdk/helm/osdk-hybrid-helm-updating-projects.adoc
+// * operators/operator_sdk/java/osdk-java-updating-projects.adoc
+
+ifeval::["{context}" == "osdk-golang-updating-projects"]
+:golang:
+:type: Go
+endif::[]
+ifeval::["{context}" == "osdk-ansible-updating-projects"]
+:ansible:
+:type: Ansible
+endif::[]
+ifeval::["{context}" == "osdk-helm-updating-projects"]
+:helm:
+:type: Helm
+endif::[]
+ifeval::["{context}" == "osdk-hybrid-helm-updating-projects"]
+:hybrid:
+:type: Hybrid Helm
+endif::[]
+ifeval::["{context}" == "osdk-java-updating-projects"]
+:java:
+:type: Java
+endif::[]
+
+:_content-type: PROCEDURE
+[id="osdk-upgrading-projects_{context}"]
+= Updating {type}-based Operator projects for Operator SDK {osdk_ver}
+
+The following procedure updates an existing {type}-based Operator project for compatibility with {osdk_ver}.
+
+ifdef::golang[]
+[NOTE]
+====
+The `go/v3` plugin is being deprecated by Kubebuilder. As a result, the Operator SDK will also be migrating to the `go/v4` plugin in a future release.
+====
+endif::[]
+
+.Prerequisites
+
+* Operator SDK {osdk_ver} installed
+* An Operator project created or maintained with Operator SDK {osdk_ver_n1}
+
+.Procedure
+
+ifdef::helm,hybrid,java[]
+* Find the `ose-kube-rbac-proxy` pull spec in the following files, and update the image tag to `v4.13`:
+endif::[]
+ifdef::ansible,golang[]
+. Find the `ose-kube-rbac-proxy` pull spec in the following files, and update the image tag to `v4.13`:
+endif::[]
++
+--
+* `config/default/manager_auth_proxy_patch.yaml`
+* `bundle/manifests/memcached-operator.clusterserviceversion.yaml`
+--
```

Comment:
I am not sure whether `--` is recommended by guidelines. Any specific reason why it's put inside a block? I assumes a simple list with file names in back ticks would work? May be I am missing something here.

Reply:
Without the use of open blocks (`--`), the indentation gets off for the multi-step procedures (makes the code block look like it's aligned with the sub-bullets):

<img width="802" alt="image" src="https://user-images.githubusercontent.com/3442316/232099245-c0602369-51db-4541-b217-c453968a4bdb.png">

With them, the code block is appropriately left-aligned with the step it's associated with:

<img width="802" alt="image" src="https://user-images.githubusercontent.com/3442316/232099479-4706ac91-82dd-4374-a8cf-b3367a727f55.png">

Without them, the single-step procedures also break:

<img width="802" alt="image" src="https://user-images.githubusercontent.com/3442316/232099683-072b4564-335a-4769-b1e6-c0a4dea8d369.png">

Reply:
There is also an example of open block usage in our repo guidelines for [Alternative footnote styling in tables](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#alternative-footnote-styling-in-tables).